graph-db/src/utils/cmr/__test__/clearScrollSession.test.js

```diff
@@ -0,0 +1,25 @@
+const { clearScrollSession } = require('../clearScrollSession')
+const nock = require('nock')
+
+process.env.CMR_ROOT = 'http://localhost'
```

Comment:
Check https://github.com/nasa/earthdata-search/blob/master/serverless/src/decodeId/__tests__/handler.test.js for a better way of setting ENV vars in tests.